src/features/usage-consolidation-accordion/usage-consolidation-content-resource.js

```diff
@@ -13,15 +13,24 @@ const propTypes = {
 };
 
 const UsageConsolidationContentResource = props => {
+  const {
+    cost,
+    costPerUse,
+    usage,
+  } = props.costPerUseData.data?.attributes?.analysis;
+
+  const noCostPerUseAvailable = !cost && !costPerUse && !usage;
+
   const customProperties = {
     columnMapping: { cost: 'ui-eholdings.usageConsolidation.summary.resourceCost' },
   };
 
   return (
     <SummaryTable
       id="resourceUsageConsolidationSummary"
-      entityType={entityTypes.TITLE}
+      entityType={entityTypes.RESOURCE}
       customProperties={customProperties}
+      noCostPerUseAvailable={noCostPerUseAvailable}
```

Comment:
Do we need the same prop for Resource Show Page? 
Looks like you delete it from summary table, and add just for Package Show Page

Reply:
In Package and Resource Summary tables we have the same logic to check that cost per use is available, but for Titles it's a little different. That's why I moved it from `<SummaryTable>` up the component tree to it's consumers

Reply:
Oh, sorry, I think that you make this changes only for package and don't look what is the file.